shapeAnalysisProject/shape_matching/hausdorff.py

```diff
@@ -0,0 +1,13 @@
+from scipy.spatial.distance import directed_hausdorff
+
+def similarity(new_image, images=[], similarity=[]):
+    print "Executing Phase 3: Calculating similarity of shapes using Hausdorff distance"
+    N = len(images)
+    for i in xrange(N):
+        similarity[i].append(directed_hausdorff(images[i], new_image)[0])
```

Comment:
Please calculate Hausdorff Distance from directed_hausdorff

I believe the formula is (though please cross-verify it)
`h1 = directed_hausdorff(X, Y)`
`h2 = directed_hausdorff(Y, X)`
`similarity[i].append(max(h1, h2))`